From a7cae24c073bd0c0da060cd6a46583102716de73 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:14 -0400
Subject: [redhat] kabi: Adding symbol set_blocksize (fs/block_dev.c)

Message-id: <6b3193fe58a130e5fe01258301480426179b603e.1410510773.git.poros@redhat.com>
Patchwork-id: 96054
O-Subject: [PATCH RHEL7.1 194/233] kabi: Adding symbol set_blocksize (fs/block_dev.c)
Bugzilla: 1062361 1071099
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

Bugzilla: 1062361,1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    Used to set the block size for the block_device when mounting a file system.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/set_blocksize
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/set_blocksize
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/set_blocksize

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/set_blocksize b/redhat/kabi/kabi-module/kabi_ppc64/set_blocksize
new file mode 100644
index 0000000..57c96bd
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/set_blocksize
@@ -0,0 +1,2 @@
+#1-
+0xffd88de1 set_blocksize vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/set_blocksize b/redhat/kabi/kabi-module/kabi_s390x/set_blocksize
new file mode 100644
index 0000000..33263be
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/set_blocksize
@@ -0,0 +1,2 @@
+#1-
+0x75eb23a7 set_blocksize vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/set_blocksize b/redhat/kabi/kabi-module/kabi_x86_64/set_blocksize
new file mode 100644
index 0000000..c49a865
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/set_blocksize
@@ -0,0 +1,2 @@
+#1-
+0xca919404 set_blocksize vmlinux EXPORT_SYMBOL
-- 
1.7.1